x-pack/plugins/security_solution/public/common/components/first_last_seen/index.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
This file was moved from host/components/first_last_seen_host to here (common). Unfortunately there were enough small changes for git to not realize it is the moved file.